### PR TITLE
LF-3480: Deploy staging contracts

### PR DESCRIPTION
**Pull request by 0xDEnYO**

### CODE REVIEW

File: .github/workflows/forge.yml [hunk 1]
```diff
@@ -32,6 +32,14 @@ jobs:
...
+        with:
+          command: forge test
+          attempt_limit: 10
+          attempt_delay: 5000
       - name: Get forge test coverage
         run: forge coverage
+
+
+
+
```
Comment (satoshi-lifi) [inline .github/workflows/forge.yml:45]: nit: blank lines

File: config/cbridge.json [hunk 2]
```diff
@@ -46,6 +46,7 @@
   "polygon": {
     "cBridge": "0x88DCDC47D2f83a99CF0000FDF667A468bB958a78",
     "messageBus": "0xaFDb9C40C7144022811F034EE07Ce2E110093fe6",
+    "cfUSDC": "0x0000000000000000000000000000000000000000",
```
Comment (satoshi-lifi) [inline config/cbridge.json:49]: nit: actually, in deploy script, it gets `cfUSDC` address only when it deploys on mainnet.
     so it doesn't need to add this.

File: script/deploy/facets/UpdateOptimismBridgeFacet.s.sol [hunk 3]
```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: UNLICENSED
 pragma solidity ^0.8.17;
 
-import { UpdateScriptBase } from "./utils/UpdateScriptBase.sol";
+import { UpdateScriptBase, console } from "./utils/UpdateScriptBase.sol";
```
Comment (satoshi-lifi) [inline script/deploy/facets/UpdateOptimismBridgeFacet.s.sol:4]: nit: unused `console` import

File: script/tasks/solidity/ApproveRefundWalletInDiamond.s.sol [hunk 4]
```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.17;
+
+import { UpdateScriptBase, console } from "../../deploy/facets/utils/UpdateScriptBase.sol";
```
Comment (satoshi-lifi) [inline script/tasks/solidity/ApproveRefundWalletInDiamond.s.sol:4]: nit: unused `console` import

File: script/tasks/solidity/ApproveRefundWalletInDiamond.s.sol [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+import { stdJson } from "forge-std/StdJson.sol";
```
Comment (satoshi-lifi) [inline script/tasks/solidity/ApproveRefundWalletInDiamond.s.sol:5]: nit: unused import

**Review by satoshi-lifi — APPROVED** [review]

File: config/cbridge.json (hunk 2 above)
Comment (satoshi-lifi) [reply config/cbridge.json:49]: image

Comment (0xDEnYO) [reply config/cbridge.json:49]: true, forgot about that...i've removed it from config

**Pull request merged**